recipes/xsd/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.40.0"
+
+
+class ConanXqilla(ConanFile):
+    name = "xsd"
+    
+    description = (
+        "XSD is a W3C XML Schema to C++ translator. It generates vocabulary-specific, statically-typed C++ mappings (also called bindings) from XML Schema definitions. XSD supports two C++ mappings: in-memory C++/Tree and event-driven C++/Parser."
+    )
+    topics = ("xsd", "xml", "c++")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://codesynthesis.com/projects/xsd/"
+    license = ("GPL-2.0","FLOSSE")
+    settings = "os", "arch", "compiler"
```

Comment:
A tool can be built with debug symbols as well.
By dismissing `build_type`, the package might contain a release or debug executable.
So I would add `build_type`.

Reply:
added build_type.